Formula/elektra.rb

```diff
@@ -0,0 +1,48 @@
+class Elektra < Formula
+  desc "Configuration Framework"
+  homepage "https://libelektra.org"
+  url "https://www.libelektra.org/ftp/elektra/releases/elektra-0.8.21.tar.gz"
+  sha256 "51892570f18d1667d0da4d0908a091e41b41c20db9835765677109a3d150cd26"
+  head "https://github.com/ElektraInitiative/libelektra.git"
+
+  option "with-qt", "Build GUI frontend"
+
+  # Build Dependencies
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+
+  # Run-Time Dependencies
+  depends_on "qt" => :optional
+  depends_on "discount" if build.with? "qt"
+
+  def install
+    cmake_args = %W[
```

Comment:
Remove `-DCMAKE_BUILD_TYPE=Release` and `-DCMAKE_INSTALL_PREFIX=#{prefix}` and use:
```ruby
args = std_cmake_args + %W[
  -DBINDINGS='cpp'
  -DTOOLS='kdb'
  -DPLUGINS='NODEP'
]
```

Reply:
I removed `-DCMAKE_BUILD_TYPE=Release`. However, if I also remove `-DCMAKE_INSTALL_PREFIX=#{prefix}`, then installing fails with the following error message:

```
Install the project...
-- Install configuration: "RelWithDebInfo"
CMake Error at cmake_install.cmake:36 (file):
  file cannot create directory: /usr/local/share/doc/elektra.  Maybe need
  administrative privileges.


make: *** [install] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting.html
```

.

Reply:
Yeah, that should go away if you use the `args = std_cmake_args + %W[` syntax, no? Homebrew's `std_cmake_args` includes `-DCMAKE_INSTALL_PREFIX=#{prefix}` as one of the default arguments passed.

Reply:
Oh, I totally overlooked the part with `std_cmake_args +` in your first comment. Thank you for the clarification.